maint-lib/ceph_version.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
Add `set -euo pipefail` here so that script fails on errors, unset variables, and failures within piped commands.

Reply:
ok